msbuild-sonarqube-runner/tools/chocolateyinstall.ps1

```diff
@@ -1,5 +1,8 @@
+$url = "https://github.com/SonarSource-VisualStudio/sonar-msbuild-runner/releases/download/1.1/MSBuild.SonarQube.Runner-1.1.zip"
+$checksum = "D9302247D6AAFAB2B76256BFC96FC8F74468B87A"
+
 ﻿Install-ChocolateyZipPackage 'msbuild-sonarqube-runner' `
```

Comment:
@yetanotherchris seems like you left the UTF-8 BOM character on this line, whereas it should be moved to the new first line - I'm taking care to fix that